crates/common/consensus/src/helpers.rs

```diff
@@ -98,3 +103,55 @@ pub fn get_voting_source(store: &Store, block_root: B256) -> Checkpoint {
         head_state.current_justified_checkpoint
     }
 }
+
+pub fn is_valid_indexed_attestation(
+    state: &BeaconState,
+    indexed_attestation: &IndexedAttestation,
+) -> bool {
+    let indices: Vec<usize> = indexed_attestation
+        .attesting_indices
+        .iter()
+        .map(|&i| i as usize)
+        .collect();
+
+    if indices.is_empty() || !is_sorted_and_unique(&indices) {
+        return false;
+    }
+
+    let pubkeys: Vec<&PubKey> = indices
+        .iter()
+        .map(|&i| &state.validators.get(i).unwrap().pubkey)
+        .collect();
+
+    let domain = match state.get_domain(
+        DOMAIN_BEACON_ATTESTER,
+        Some(indexed_attestation.data.target.epoch),
+    ) {
+        Ok(domain) => domain,
+        Err(_) => return false,
+    };
+
+    let signing_root = compute_signing_root(&indexed_attestation.data, domain);
+
+    let sig =
+        blst::min_pk::Signature::from_bytes(&indexed_attestation.signature.signature).unwrap();
+
+    let publickeys: Vec<blst::min_pk::PublicKey> = pubkeys
+        .iter()
+        .filter_map(|key| blst::min_pk::PublicKey::from_bytes(&key.inner).ok())
+        .collect();
+
+    let verification_result = sig.fast_aggregate_verify(
+        true,
+        signing_root.as_ref(),
+        domain.as_ref(),
+        publickeys.iter().collect::<Vec<_>>().as_slice(),
+    );
```

Comment:
```suggestion
pub const DST: &[u8] = b"BLS_SIG_BLS12381G2_XMD:SHA-256_SSWU_ROPOP";
    let verification_result = sig.fast_aggregate_verify(
        true,
        signing_root.as_ref(),
        DST,
        publickeys.iter().collect::<Vec<_>>().as_slice(),
    );
```
Shouldn't it be this?

Reply:
@varun-doshi I think this should be applied